testdata/includes/module1/Taskfile.yml

```diff
@@ -0,0 +1,9 @@
+version: '2'
```

Comment:
Those test files should all be in version 3. Also, we should disallow setting a dir if version is 2. Just returning an error if version is 2 and dir is not `""` is fine. This should be on `Setup()` like the other version validations.